### DISCUSSION TITLE
Multiple Track Saving

### DISCUSSION THREAD

**Discussion by particl3s** [OpenRCT2/OpenRCT2#19092] [Feature requests] [answered]
Many rides in OpenRCTII are comprised of trackitecture, merging and other things such as shoestring injection bays. Might I suggest feature saving of multiple rides in the 'save track' section. That is to say saving more than one track into **one** since currently only scenery objects can be packed in.

![This is an image](https://i.imgur.com/rCNvuGm.png)

Often rides of mine have 7 or 8 additional tracks adjacent to the main component, without which it is not the same ride. And lastly invisibility of track pieces need recursive properties, such as position or visibility inside the tile inspector. Since many elements conflict...thanks!

**Comment by spacek531** [comment, marked as answer]
Saving of multiple tracks requires a new track design format which has been several years coming.
